config/keycloak/realm-import.yaml

```diff
@@ -75,47 +75,47 @@ data:
           ],
           "server1.mcp.local": [
             {
-              "name": "test_greet"
+              "name": "greet"
```

Comment:
If the `x-mcp-toolname` continues to carry `<server-prefix>_<tool-name>`, won't this break authorisation at the `tools/call` request? https://github.com/guicassolato/mcp-gateway/blob/a2bd55563884bd5c8c9671a520973377a1a117d3/config/samples/oauth-token-exchange/tools-call-auth.yaml#L41

Reply:
i changed it to be the actual tool name as I think from an admin perspective it makes more sense to use the actual tool name